platforms/ide/tooling-api/src/main/java/org/gradle/tooling/internal/protocol/InternalProblemSummary.java

```diff
@@ -18,7 +18,11 @@
 
 import org.gradle.api.NonNullApi;
 
+import java.io.Serializable;
+
 @NonNullApi
-public interface InternalProblemAggregationDetails extends InternalProblemDetails {
```

Comment:
❌ Another change that may break cross-version compatibility.

Reply:
Can you explain a bit more?